src/components/utils/ask-text-dialog.tsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { useIntl } from 'react-intl';
+import { Dialog, DialogActions, DialogContent, DialogTitle } from '@mui/material';
+import Button from '@mui/material/Button';
+import React, { FunctionComponent, useEffect } from 'react';
+import { useValidNodeName } from './inputs/input-hooks';
+import { useSelector } from 'react-redux';
+import Alert from '@mui/material/Alert';
+import { CancelButton } from '@gridsuite/commons-ui';
+import { AppState } from '../../redux/reducer';
+
+interface AskTextDialogProps {
+    title: string;
+    value: string;
+    show: boolean;
+    onValidate: (newValue: string) => void;
+    onClose: () => void;
+}
```

Comment:
you've removed the existing comment giving some explanations for this component
don't you think we should keep it ?

Reply:
i put the description back